### ISSUE TITLE
network-ip-availabilities api

### ISSUE THREAD

**Issue by ligaofeng0901** [ContainX/openstack4j#1201]
The API `/v2.0/network-ip-availabilities/` in `Networking API v2.0`, The extension network-ip-availability allows users to list and show the network IP usage stats of all networks or of a specified network.
Is this API uncompleted？If is so, how can I extend?

**Comment by champagne** [comment]
Yes, we can see the API from [Network IP availability and usage stats](https://developer.openstack.org/api-ref/network/v2/index.html?#network-ip-availability-and-usage-stats). It could help us to know the IP usage stats.
